.github/workflows/close-stale-issues.yml

```diff
@@ -0,0 +1,55 @@
+name: Close stale issues
+
+# Controls when the action will run.
+on:
+  schedule:
+  # Uses UTC so it runs at 10PM PDT
+  - cron: "0 6 * * *"
+
+jobs:
+  cleanup:
+    runs-on: ubuntu-latest
+    name: Stale issue job
+    steps:
+    - uses: aws-actions/stale-issue-cleanup@v3
+      with:
+        issue-types: issues
+      
+        # Setting messages to an empty string will cause the automation to skip
+        # that category
+        ancient-issue-message: |
+          This issue has not received any attention in 1 year.
+          If you want to keep this issue open, please leave a comment below and auto-close will be canceled.
+        stale-issue-message: |
+          This issue has not received a response in 14 days.
+          If you want to keep this issue open, please leave a comment below and auto-close will be canceled.
+        stale-pr-message: |
+          This PR has not received a response in 14 days.
+          If you want to keep this issue open, please leave a comment below and auto-close will be canceled.
+
+        # These labels are required
+        stale-issue-label: blocked/close-if-inactive
+        exempt-issue-labels: no-autoclose, stage/need-attention
+        stale-pr-label: blocked/close-if-inactive
+        exempt-pr-labels: no-autoclose, type/feature
+        response-requested-label: blocked/more-info-needed
+
+        # Don't set this to not apply a label when closing issues
+        closed-for-staleness-label: closed-for-inactive
```

Comment:
nit: `closed-for-inaction` or `closed/inactive` might be more correct

Reply:
I would go with: `stage/closed-for-inactivity`